src/archiving/directory_archivable.e

```diff
@@ -36,16 +35,20 @@ feature -- Status
 
 	header: TAR_HEADER
 			-- Header that belongs to the payload
+		local
+			f: like directory
 		do
+			f := directory
```

Comment:
use a local ... here this is not critical, but sometime in loops .. it can be significant.
attribute access vs local variable access.